core/arch/arm/kernel/stmm_sp.c

```diff
@@ -905,7 +908,7 @@ static bool spm_handle_direct_req(struct thread_svc_regs *regs)
 
 static bool spm_handle_svc(struct thread_svc_regs *regs)
 {
-	unsigned long int *a0 = 0;
```

Comment:
The `int` was introduced in the previous commit, it should probably never had been there.